server/channels/store/sqlstore/scheduled_post_store.go

```diff
@@ -239,3 +245,22 @@ func (s *SqlScheduledPostStore) toUpdateMap(scheduledPost *model.ScheduledPost)
 		"ErrorCode":   scheduledPost.ErrorCode,
 	}
 }
+
+func (s *SqlScheduledPostStore) Get(scheduledPostId string) (*model.ScheduledPost, error) {
+	query := s.getQueryBuilder().
+		Select(s.columns("")...).
```

Comment:
Does this behave differently from `Select("*")`?

Reply:
We don't use `select *` queries in server (except for very old SQL methods) as it causes issues wit upgrades in cluster. What happens is that on the node with old server version running, the model has, say, 5 fields, and the new node added a migration adding a 6th field as well. On the old server, if it now did a `select *`, gorm would receive 6 fields     per row and would be asked to map it to 5 fields in the model, and it will fail doing so. So this is why we stopped using select * and instead always explicitly specify all the fields we want. We're doing the same in plugins as well - boards, survey, playbooks, calls etc.